src/main/java/org/broadinstitute/dsm/db/dao/participant/data/ParticipantDataDao.java

```diff
@@ -0,0 +1,145 @@
+package org.broadinstitute.dsm.db.dao.participant.data;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.broadinstitute.ddp.db.SimpleResult;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.db.dto.participant.data.ParticipantDataDto;
+
+public class ParticipantDataDao implements Dao<ParticipantDataDto> {
+
+    private static final String SQL_PARTICIPANT_DATA_BY_PARTICIPANT_ID = "SELECT * FROM ddp_participant_data WHERE ddp_participant_id = ?";
```

Comment:
I am not sure I like selecting `*` instead of specific columns…

Reply:
do you mean to just list all the columns instead of `*`?